src/plugins/intel_cpu/src/emitters/snippets/x64/kernel_executors/brgemm_copy_b.cpp

```diff
@@ -0,0 +1,285 @@
+    mov(abi_param1, reinterpret_cast<uintptr_t>(kernel.get()));
+
+    add_offset(src_reg, offset_in); // abi_param2
+    add_offset(tr_src_reg, offset_out); // abi_param3
+    if (conf.is_with_comp())  // abi_param4
+        add_offset(comp_reg, offset_comp);
+    else
+        mov(comp_reg, reinterpret_cast<uintptr_t>(nullptr));
+
+#ifdef _WIN32
+    // Note: ABI requires that the remaining parameters (except the first for) are pushed to the stack in right-to-left order
+    //  Shadow space will be allocated inside internal_call_rsp_align()
+    push(K);
+    push(N);
+#else
+    mov(abi_param5, N);
+    mov(abi_param6, K);
+#endif
+
+    safe_internal_caller.call(rbp);
+
+#ifdef _WIN32
+    static constexpr int gpr_size = 8;
+    add(rsp, gpr_size * 2);
+#endif
+}
+
+void BrgemmCopyBKernel::execute(matmul::jit_brgemm_matmul_copy_b_t* kernel, const void* src, const void* dst, const void* comp, size_t N, size_t K) {
+    auto ctx = matmul::jit_brgemm_matmul_copy_b_t::ctx_t();
+    ctx.current_N_blk = N;
+    ctx.src = src;
+    ctx.tr_src = dst;
+    ctx.compensation_ptr = comp;
+    ctx.zp_a_compensation_ptr = nullptr;
+    ctx.zp_a_neg_value_ptr = nullptr;
+    ctx.current_K_start = 0;
+    ctx.current_K_iters = K;
+
+    OV_CPU_JIT_EMITTER_ASSERT(kernel, "Kernel hasn't been created");
+    (*kernel)(&ctx);
+}
+
+BrgemmCopyBKernelExecutor::BrgemmCopyBKernelExecutor(ov::intel_cpu::MultiCacheWeakPtr kernel_cache, BrgemmCopyBKernelConfig config)
+    : CPUKernelExecutor<BrgemmCopyBKernelConfig, BrgemmCopyBKernel>(std::move(kernel_cache), std::move(config)) { }
+
+std::shared_ptr<BrgemmCopyBKernel> BrgemmCopyBKernelExecutor::compile_kernel(const BrgemmCopyBKernelConfig& config) const {
+    std::shared_ptr<BrgemmCopyBKernel> compiled_kernel = std::make_shared<BrgemmCopyBKernel>();
+    // BrgemmCopyB is not executable - nothing to compile
+    if (!config.is_empty()) {
+        compiled_kernel = std::unique_ptr<BrgemmCopyBKernel>(new BrgemmCopyBKernel(config));
+        OV_CPU_JIT_EMITTER_ASSERT(compiled_kernel, "compiled kernel is nullptr");
+        compiled_kernel->create_kernel();
+    }
+
+    return compiled_kernel;
+}
+
+void BrgemmCopyBKernelExecutor::update_config(const ov::snippets::lowered::ExpressionPtr& expr,
+                                              const ov::snippets::lowered::LinearIRCPtr& linear_ir,
+                                              BrgemmCopyBKernelConfig& config) const {
+    const auto planar_shape = ov::snippets::utils::get_planar_vdims(expr->get_input_port(0));
+    const auto N = *planar_shape.rbegin();
+    const auto K = *++planar_shape.rbegin();
+
+    const auto& in_subtensor = ov::snippets::utils::get_projected_subtensor(expr->get_input_port(0));
+    const auto N_blk = *in_subtensor.rbegin();
+    const auto K_blk = *++in_subtensor.rbegin();
+    OV_CPU_JIT_EMITTER_ASSERT(N_blk <= N && K_blk <= K, "BrgemmCopyB has incompatible subtensor dimensions");
```

Comment:
Do we support blocking for this executor? I am asking because in brgemm executor, which supports blocking, we have logic for actual subtensors computations based on loops.